### ISSUE TITLE
Create Press processes page

### ISSUE THREAD

**Issue by naushinthomson** [elifesciences/schematron-gitbook#186]
- [ ] Create page 
- [ ] Add description of press processes including press update meetings/emails
- [ ] Melissa approved page
- [ ] Fred approved page
- [ ] James approved page
- [ ] Naushin approved page
- [ ] Claudia approved page
- [ ] Share page with Exeter and invite feedback

### Definition of done
- [ ] Page revised after team comments
- [ ] Team reviewed and approved page?

**Comment by griffithsc** [comment]
Hi everyone, the press processes page is now ready to [review](https://elifesciences.gitbook.io/productionhowto/-M1eY9ikxECYR-0OcnGt/press-processes). Thanks for your feedback! :) @Melissa37 @JGilbert-eLife @naushinthomson @FAtherden-eLife

**Comment by FAtherden-eLife** [comment]
Thanks @griffithsc. 

> If it due to go out to the author later in the week, it should be scheduled for the following Tuesday. For instance, in the screenshot above, the proof goes out to the authors on a Wednesday (7th July), so should be scheduled for the Tuesday after next (20th July).

If you mean the 20th in this sentence (instead of the 13th) can we change "following Tuesday" to some other clearer phrase? 

also typo  => If **it's** due

**Comment by FAtherden-eLife** [comment]
There's no mention of time - can you add something about that (the fact that schedule articles for different times in the day, usually depending on the corresponding authors institutional time zone).

**Comment by Melissa37** [comment]
>If the article is due to reach Author Review on either a Monday or a Tuesday, then it can be scheduled for the nearest Tuesday. 

I read that to mean you can schedule it for THAT Tuesday, not a later one, which feels like it's cutting it a bit fine.

**Comment by FAtherden-eLife** [comment]
Could you explain what to do if a VoR is ready to send to Continuum and the PoA has already been scheduled for publication for a future date?

**Comment by Melissa37** [comment]
>As can be seen in the screenshot below, the email should clearly list all of the articles that have been scheduled, specifying the time and date and whether they have any related content such as an Insight.

Is there a chance co-subs are press released? Should they be mentioned?

**Comment by FAtherden-eLife** [comment]
Please could you explain what Emily does in relation to Insight articles for Research articles getting Press, and how that might affect us?

**Comment by FAtherden-eLife** [comment]
Can we add a link to our media policy somewhere, and how to respond to authors who ask about embargos etc.?

**Comment by Melissa37** [comment]
Thanks! Looks good :-)

Only other minor comment - we use Oxford comma I think (a comma in front to the and in a list of items in a sentence)

**Comment by FAtherden-eLife** [comment]
Is it worth mentioning somewhere that the eLife (or freelance) written press packs are here - https://elifesciences.org/for-the-press?

**Comment by FAtherden-eLife** [comment]
I think we should change `Scheduling articles in Kriya` to something else. Maybe `Marking articles as Press in Kriya` or something similar. The articles aren't scheduled for publication in Kriya, that's done in Continuum.

**Comment by FAtherden-eLife** [comment]
Nice work, thanks @griffithsc.

**Comment by JGilbert-eLife** [comment]
It might be worth slightly loosening the language at the start. We have occasionally published articles on days other than Tuesday when there were unusual circumstances. Maybe 'almost always'? And say that if the authors insist on another date, that can be accommodated?

**Comment by JGilbert-eLife** [comment]
Also, Emily's title is officially 'Media Relations Manager' :)

**Comment by JGilbert-eLife** [comment]
Please could you make it clear that 'chronological' in the context of the press meetings means in terms of press date? Articles are showing in chronological order in Kriya but it's by acceptance date.

**Comment by JGilbert-eLife** [comment]
Is the screenshot of the template cut off at the top?

**Comment by JGilbert-eLife** [comment]
Minor point, but the email below that is from Jo so it's probably better if the text explains that both Emily and Jo/freelancers can send hold messages.

**Comment by JGilbert-eLife** [comment]
We've bolded links on other pages - would you mind doing the same for consistency? It makes them slightly more obvious.

**Comment by JGilbert-eLife** [comment]
I agree about making time factors more prominent; this has a big impact on when we're judging if something needs to be pushed back and can also impact on our judgement for when an article can be scheduled. I also think that things might need to be a bit clearer about how we estimate the dates we give to Emily. I've never bothered with the due dates in Kriya so much as an estimate for how long I expect the remaining stages to take. For example, I'd usually assume a no-digest article that's waiting for its decision letter would need about a week and a half to get to publication (best case). If it's already with the author, that's going to usually need less time, but not if there's been communication with the author that indicates a delay. And so on.

**Comment by JGilbert-eLife** [comment]
Thanks @griffithsc!

**Comment by FAtherden-eLife** [comment]
I recently added a Schematron check for publication dates in the future (at date of validating) - `press-pub-date-check`. It fires a warning if the pub date is in the future and is not on a Tuesday. Is it worth adding to this page, or to a different one that covers publication dates?

**Comment by naushinthomson** [comment]
>The Production team has a weekly meeting with Emily to schedule and to track the progress of upcoming press articles. 

suggest changing to 

>The Production team has a weekly meeting with eLife's press officer, Emily Packer to schedule and to track the progress of upcoming press articles.

**Comment by naushinthomson** [comment]
>When checking the status of press articles, if any of those marked for next Tuesday are still at the Author Review stage, they do not need to be pushed back at this point, and Emily can be updated on these in the Press Update email on Monday.

I think a bit more detail is needed here - we may have already corresponded with the authors and they may, for example, have requested extensive edits and a second round of review. In that case, it might be better to push the press date to the following Tuesday.

**Comment by naushinthomson** [comment]
Would it be better to structure the press meeting section in an ordered list? Something like: 

- Confirm scheduled articles
- Check articles in progress
- Estimate press dates for upcoming articles

**Comment by naushinthomson** [comment]
> Is there a chance co-subs are press released? Should they be mentioned?

Yes, it's definitely worth covering this - sometimes one press article needs to be pushed back because its co-sub is delayed.

**Comment by naushinthomson** [comment]
>It is advisable to take the same course of action when an author enquires about a publication date, as often this means that they are considering press. 

I'm not sure this is necessary actually - maybe something we need to discuss in the production meeting?

**Comment by griffithsc** [comment]
Hi everyone, [this page](https://elifesciences.gitbook.io/productionhowto/-M1eY9ikxECYR-0OcnGt/press-processes) is ready to review again. It has been edited to incorporate your (helpful) feedback, to add in info about PoAs on hold, and the press-pub-date-check schematron check. Thanks! @Melissa37 @JGilbert-eLife @naushinthomson @FAtherden-eLife

**Comment by naushinthomson** [comment]
@griffithsc Sorry for the delay - I think everything looks good, just one final thing, for the 'Estimate press dates for upcoming articles' section I think it would be better to remove the part about consulting due dates altogether, and instead use the information we have about how long each stage should take instead. So, you can say something like:

The estimated due date will depend on the stage an article is currently at. Unless there are problems, this is how long articles spend at each stage:
Pre-editing - 2 days
Copyediting - 2 days 
Typesetter QA - 2 days
Waiting for assets if relevant - check due date of digest on tracker
Author review - 2 days
Publisher review - 1 day

And then keep the part from 'If an article has been marked...'
Also at the end of that section I think a sentence on Decision letters would also be good, just to say if an article is in the waiting for assets queue and only waiting for a decision letter this will usually be loaded within a couple of days.
